### PR TITLE
restaurant website added.

### PR DESCRIPTION
**Pull request by jugeshraghav**
a website with database.

#104

### CODE REVIEW

**Comment by chicken-biryani** [comment]
Which issue number does this deal with?

**Comment by chicken-biryani** [comment]
Add your project Title and description in index.md

File: Restaurant-website/index.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            <input type="submit" name="button" value="SEND MESSAGE"><br>
+        </form>
+    </div>
+<!-------Footer-------------------------------------------------------------------------------------------------------------------------------->
+<div id="footer">
+    <span id="footer-content">Powered by @jugesh-raghav...</span>
+</div>
+</body>
+
+</html>
```
Comment (chicken-biryani) [inline Restaurant-website/index.html:138]: Add a new line at the end of file.

Use an HTML Beautifier.

File: Restaurant-website/index.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+visitor3.save();
+res.redirect("/");
+})
+
+
+
+
+app.listen(3000,(req,res)=>{
+    console.log("server running at port 3000");
+})
```
Comment (chicken-biryani) [inline Restaurant-website/index.js:77]: Add a new line at the end of the file.
Use a JS beautifier.

File: Restaurant-website/public/css/style.css [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+    padding:20px;
+}
+#footer-content{
+    color:white;
+    letter-spacing:5px;
+}
+/*****horizontal rule**********/
+hr{
+    color:grey;
+}
```
Comment (chicken-biryani) [inline Restaurant-website/public/css/style.css:209]: Add a new line at the end of file. Use a css beautifier.

**Review by chicken-biryani — COMMENTED** [review]
1)Add project name in form of link and description in index.md

2) Add a small README.md file to the project.

Contents of the README:

-Project overview.
-Use of the project.
-Stack of your project.
-How to set up / run your project.
-Screenshots of the pages/output.
(A short description under each screenshot... Describing the page a little bit)
This would help visitors/developers in general

3)Please use the following tools to increase the readability of your README / Scripts before creating a pull request.
It is a good practice.

Markdown - https://codebeautify.org/markdown-formatter
HTML - https://codebeautify.org/htmlviewer
CSS - https://codebeautify.org/css-beautify-minify
JS - https://codebeautify.org/jsviewer

Use this PR as a reference :
#93

**Comment by jugeshraghav** [comment]
hii @Ayushparikh-code  i have updated the readme file. please look at it.

File: Restaurant-website/index.html (hunk 1 above)
Comment (chicken-biryani) [reply Restaurant-website/index.html:138]: @jugeshraghav Add a new line at the end of file.

Use an HTML Beautifier.

File: Restaurant-website/index.js (hunk 2 above)
Comment (chicken-biryani) [reply Restaurant-website/index.js:77]: > Add a new line at the end of the file.
> Use a JS beautifier.

@jugeshraghav

File: Restaurant-website/public/css/style.css (hunk 3 above)
Comment (chicken-biryani) [reply Restaurant-website/public/css/style.css:209]: > Add a new line at the end of file. Use a css beautifier.

@jugeshraghav

File: Restaurant-website/readme.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# Welcome 🖐 to the Restaurant Website
+It is a one pager Restaurant Website with an integrated databse.
+
+
```
Comment (chicken-biryani) [inline Restaurant-website/readme.md:4]: Add a "Use of the project" Section.

File: Restaurant-website/readme.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+- go to any browser and type the following in the search bar
+```
+localhost:3000
+```
+
+<br>
+
+
+
+## Happy Coding!
```
Comment (chicken-biryani) [inline Restaurant-website/readme.md:58]: Add screenshots

**Review by chicken-biryani — COMMENTED** [review]
Add your project's name and description in 'index.md'

**Comment by harshita2216** [comment]
@jugeshraghav, any updates on this?

File: Restaurant-website/readme.md (hunk 4 above)
Comment (jugeshraghav) [reply Restaurant-website/readme.md:4]: what should be added in this section.

**Comment by jugeshraghav** [comment]
I would add the screen shots else "add a new line at the end of the file." i didn't get it

**Comment by chicken-biryani** [comment]
@jugeshraghav The last line of all your files should be a blank line.
So just press enter and save the file.

Make sure there is just one blank line at the end and not more.

**Comment by jugeshraghav** [comment]
I have done all the necessary changes. please review my PR. @chicken-biryani @harshita2216

**Pull request closed without merging**